yt/yt/library/auth_server/auth_cache-inl.h

```diff
@@ -47,7 +48,10 @@ TFuture<TValue> TAuthCache<TKey, TValue, TContext>::Get(const TKey& key, const T
                     auto guard = Guard(entry->Lock);
                     entry->Updating = false;
 
+                    const auto& Logger = AuthLogger;
+
                     if (transientError) {
+                        YT_LOG_DEBUG(value, "Skipping transient error while updating authentication cache entry (EntryAddress: %v, ErrorCode: %v)", entry.Get(), value.GetCode());
```

Comment:
Why do we need to log entry address?

Reply:
To be able to match differrent log entries(are they about the same entry?) and debug cases similar to "why cache with TTL of 5 seconds returns stale value for 30 minutes"
The best way would be to log key, but they are sensitive now ('ytct-abcd-efghji')
So as a tradeoff I decided to log entry address